### DISCUSSION TITLE
Recruiting

### DISCUSSION THREAD

**Discussion by g2ktcf** [zencart/zencart#4819] [General]
One of the major things @torvista pointed out in the original "Zen Card is Dead" post, was our number of active developers is now down into the single digits.  I wanted to start this discussion whose sole purpose is to recruit development help.  I am not anywhere close of one of these so I am simply serving as a helper to get this going.  In my day job, I am a Project Manager for a Houston, TX chemical plant.  Everything I do is about allocating resources so this is something I know a little bit about. 

1.  New coders ->  How do we bring new developers?  New would also imply training (heck, I might could help more with some training!)  New would also be "how to protect us from those we do not know"  The last thing we want is a fox in the hen house, fraud wise.
2.  We have lost some seriously needed support from others.  Why?  Can we fix this?  Are they swamped?  Are they upset at someone here?  I will gladly reach out to those with an olive branch if it helps.
3.  What else can we do?  

There appears to be LOADS to do now and everything actively being discussed requires even more support.

Thanks!
Chris

**Comment by torvista** [comment]
Current Users to Contributors
One thing is new users, who will be their nature will unlikely to be contributors in the short term.
The most likely new contributors are be current users.

What can be done to turn users into contributors?
Pointing them at a GitHub list to fix bugs, I do not see as being likely to generate interest as much as fear.
The transition ZC157 to ZC158, as I've said elsewhere, would probably be best done as a site rebuild (incorporating mods into core ZC158) and so users will have to get their hands dirtier than usual.
This seems an ideal time to promote the idea that, "if you need it, maybe others do too", to get them to finally set up a development site (with detailed instructions as to best practices) and to explain clearly how GitHub works, how to contribute and what is needed/expected, to encourage them to contribute at least the reasons for their mods and code.

It took me a long time to get over my worry about how to contribute, did I know what I was doing, getting in knots with Git, was the code good enough, fear of getting it wrong/criticism etc. Some might say it should have taken longer. 

Another reason to invest in more hand-holding for the upgrade process is that it is the pinch-point when people think "why is this so damned difficult" and look for alternatives in the market/jump ship.
